libraries/AP_HAL_SITL/SITL_cmdline.cpp

```diff
@@ -243,6 +243,10 @@ void SITL_State::_parse_command_line(int argc, char * const argv[])
             break;
         }
     }
+    if (sitl_model == nullptr) {
```

Comment:
Oh, never mind. The `sitlState` variable is static, and, thus, zero-initialized.